src/navigator/root-stack.tsx

```diff
@@ -175,6 +172,7 @@ export const RootStackScreen = () => {
         </CurrentRouteNameContext.Provider>
       </PortalProvider>
 
+      {isSplash && <SplashModal />}
```

Comment:
why did you create your own "Splash screen" and not using the real, native Splash screen?

Reply:
react-native-bootsplash deprecated `show()` method in newer version of package and its not working in old version with newer RN package